proj/cmake/libcinder_target.cmake

```diff
@@ -24,11 +24,12 @@ target_include_directories( cinder SYSTEM BEFORE INTERFACE ${CINDER_INCLUDE_SYST
 target_include_directories( cinder BEFORE PRIVATE ${CINDER_INCLUDE_USER_PRIVATE} )
 target_include_directories( cinder SYSTEM BEFORE PRIVATE ${CINDER_INCLUDE_SYSTEM_PRIVATE} )
 
-target_link_libraries( cinder PUBLIC ${CINDER_LIBS_DEPENDS} )
+target_link_libraries( cinder PUBLIC ${CINDER_LIBS_DEPENDS}  )
+target_link_libraries( cinder LINK_PRIVATE Ws2_32.lib  )
```

Comment:
I think this line should be removed since a) it would 'leak' to non-windows platforms and b) it has been integrated in `platform_msw.cmake` as part of `CINDER_LIBS_DEPENDS` [here](https://github.com/cinder/Cinder/pull/1594/files/ead79ef496ef382b417fe9ffe239afcc57c358d3..e8d2c322b5e3332e4004358a4ee6808b8b725716#diff-034b9f3e1e4aa1f6f60463fa1c03ba8aR65)?